### PR TITLE
WIP 22549 Experimental DelayIdleTicker

### PR DESCRIPTION
**Pull request by bencoman**

### CODE REVIEW

File: src/Kernel/Delay.class.st [hunk 1]
```diff
@@ -38,26 +38,29 @@ Delay class >> anyActive [
...
 { #category : #settings }
 Delay class >> delaySchedulerClass [
 	^Scheduler class
 ]
 
 { #category : #settings }
 Delay class >> delaySchedulerClass: aSchedulerClass [ 
-	| newScheduler oldScheduler |
 	self delaySchedulerClass = aSchedulerClass ifTrue:[ ^self ].
-	newScheduler := aSchedulerClass new.
```
Comment (bencoman) [inline src/Kernel/Delay.class.st:50]: This was extracted into delayScheduler: above

**Comment by Ducasse** [comment]
Hi Ben 
we are getting back on track. 
Could you tell us if you have some cycles for this fix? in case something wrong happen after integration :)

Stef

**Comment by Ducasse** [comment]
Hi Ben 
we are getting back on track. 
Could you tell us if you have some cycles for this fix? in case something wrong happen after integration :)

Stef

**Comment by bencoman** [comment]
Sorry I didn't notice the stale tagging.  This is still active.

**Comment by bencoman** [comment]
actually you can leave it closed.  I'll submit a new pull request when I have some cycles to deal with it.

**Pull request closed without merging**